eox_nelp/processors/tests/xapi/event_transformers/tests_initialized_events.py

```diff
@@ -0,0 +1,76 @@
+"""This file contains all the test for the initialized_events.py file.
+
+Classes:
+    InitializedCourseTransformerTestCase: Tests cases for InitializedCourseTransformer class.
+"""
+from django.test import TestCase
+from mock import Mock, patch
+from tincan import ActivityDefinition, LanguageMap
+
+from eox_nelp.edxapp_wrapper.event_routing_backends import constants
+from eox_nelp.processors.xapi import constants as eox_nelp_constants
+from eox_nelp.processors.xapi.event_transformers import InitializedCourseTransformer
+
+
+class InitializedCourseTransformerTestCase(TestCase):
+    """Test class for InitializedCourseTransformer class."""
+
+    def setUp(self):
+        """Setup common conditions for every test case"""
+        self.get_data_mock = Mock()
+        self.get_object_iri_mock = Mock()
+
+        InitializedCourseTransformer.get_data = self.get_data_mock
```

Comment:
What is the advantage of this ?

Reply:
That maybe could be more clear to the reader the direct assignation rather than if you assign self.get_data, underneath you are assigning InitializedCourseTransformer.get_data too.